README.md

```diff
@@ -1,24 +1,3 @@
-# Duke project template
+# Mortal Reminder Chatbot Project!
```

Comment:
Overall the code is very clean, organised and adheres with the coding standards. Good job!